traffic_stops/settings/base.py

```diff
@@ -357,3 +358,5 @@ def __init__(self, tz_name=None):
 
 # Contact us Email
 CONTACT_US_EMAILS = os.getenv("CONTACT_US_EMAILS", "forwardjustice-team@caktusgroup.com").split(":")
+
+CKEDITOR_BASEPATH = "/static/ckeditor/ckeditor/"
```

Comment:
Need to check if the deployment runs `collectstatic`

Reply:
@copelco If hard coding the path to `static` ok in prod? I see the Dockerfile sets a `UWSGI_STATIC_MAP` to something different. Or is that totally unrelated?

Reply:
I think this should work!